x-pack/plugins/cloud_security_posture/server/create_indices/create_processor.ts

```diff
@@ -59,3 +46,65 @@ export const createPipelineIfNotExists = async (
   }
   return false;
 };
+
+export const getPipelineById = (id: string): PipelineIngest => {
+  switch (id) {
+    case CSP_INGEST_TIMESTAMP_PIPELINE: {
+      return findingsPipeline;
+    }
+    case CSP_LATEST_INGEST_TIMESTAMP_PIPELINE: {
+      return latestFindingsPipeline;
+    }
+    default:
+      throw new Error('No ingest pipeline was found for this Id');
+  }
+};
+
+export interface PipelineIngest {
+  id: string;
+  description: string;
+  on_failure: Array<{ set: { field: string; value: string } }>;
+  processors: Array<{ set: { field: string; value: string } }>;
+}
+
+const findingsPipeline = {
```

Comment:
this is not "findingsPipeline", it used the score index,

anyway, since pipeline not have to serve specific index I would give it more general name and not including the index it serves